core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalStatsTests.java

```diff
@@ -32,13 +32,12 @@
     @Override
     protected InternalStats createTestInstance(String name, List<PipelineAggregator> pipelineAggregators,
                                                Map<String, Object> metaData) {
-        long count = randomIntBetween(1, 50);
-        double[] minMax = new double[2];
-        minMax[0] = randomDouble();
-        minMax[0] = randomDouble();
-        double sum = randomDoubleBetween(0, 100, true);
-        return new InternalStats(name, count, sum, minMax[0], minMax[1], DocValueFormat.RAW,
-            pipelineAggregators, Collections.emptyMap());
+        long count = randomIntBetween(0, 50);
+        double min = randomDoubleBetween(-1000, 1000, false);
+        double max = randomDoubleBetween(-1000, 1000, false);
+        double sum = randomDoubleBetween(-1000, 1000, false);
```

Comment:
I increased the range already but this leads to rounding errors in the reduce logic so we need to be more lenient with the delta here then: https://github.com/elastic/elasticsearch/pull/24212/files/ac7bfcc469014fb376fbbbd7396fc726d55450c8#diff-3d8a3c85fc5c06b0859a2a4749fb834dL61